blog/docs/articles/demystifying-activator-on-path.md

```diff
@@ -0,0 +1,280 @@
+    Also, in the proxy mode, the SKS controller will pick up the number of activators and configure an equal number of endpoints for the revision's [public service](https://github.com/knative/serving/blob/main/docs/scaling/SYSTEM.md#data-flow-examples).
+    In combination with the networking setup done by a Knative networking component (driven by the Ingress resource), this is roughly the end-to-end networking setup that needs to happen for a ksvc to be ready to serve traffic.
+    The Knative networking component can be any of the following: net-istio, net-kourier, net-contour, and net-gateway-api.
+
+## Capacity and Operation Modes in Practice
+
+As described earlier, the activator will be removed from the path, assuming enough capacity is available. 
+Let's see how this capacity is calculated, but before that, let's introduce two concepts: the `panic` and `stable` windows.
+The `panic` window is the time duration that denotes a lack of capacity to serve the traffic. It happens usually with a sudden burst of traffic.
+The condition that describes when to enter the panic mode and start the panic window is:
+
+```
+dppc/readyPodsCount >= spec.PanicThreshold
+```
+where
+```
+dppc := math.Ceil(observedPanicValue / spec.TargetValue)
+```
+
+The `dppc` stands for desired panic pod count and expresses what autoscaler needs to achieve in panic mode.
+The target value is the utilization in terms of concurrency that the autoscaler aims for and that is calculated as 0.7*(revision_total).
+Revision total is the maximum possible value of the scaling metric that is permitted on the pod and defaults to 100 (container concurrency default).
+The value 0.7 is the utilization factor for each replica and when that is reached we need to scale out.
+
+**Note:** if the KPA metric Requests Per Second(RPS) is used then the utilization factor is 0.75.
+
+The `observedPanicValue` is the calculated average value seen during the panic window for the concurrency metric.
+The panic threshold is configurable (default 2) and expresses the ratio of desired versus available pods.
+
+After we enter panic mode in order to exit we need to have enough capacity for a period that is equal to the stable window size.
+That also means that the autoscaler will try to get enough pods ready in order to increase the capacity.
+Also, note here that when operating outside the panic mode the autoscaler does not use `dpcc` but a similar quantity:
+`dspc := math.Ceil(observedStableValue / spec.TargetValue)` which is based on metrics during the stable period.
+
+To quantify the idea of enough capacity and to deal with bursts of traffic we introduce the notion of the Excess Burst Capacity(EBC) that needs to be >=0.
+It is defined as:
+
+```
+EBC = TotalCapacity - ObservedPanicValue - TargetBurstCapacity(TBC).
+```
+
+`TotalCapacity` is calculated as ready_pod_count*revision_total. The default TBC is set to 200. 
+
+For the Knative Autoscaler it holds the following.
```

Comment:
unclear what the following refers to here... I'm assuming the important box, but I also usually assume those boxes are digressions from the main text...